user-service/src/main/java/com/innowise/userservice/entity/CardInfo.java

```diff
@@ -0,0 +1,55 @@
+package com.innowise.userservice.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+/**
+ * Entity class which represents card information.
+ * <p>
+ * This class maps to the "card_info" table in the database.
+ * Contains card details associated with a specific user.
+ * </p>
+ */
+@Entity
+@Table(name = "card_info")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CardInfo {
+    /** Unique identifier for the card. Automatically generated by the database. */
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    /** The user who owns this card. Multiple cards can belong to a single user. */
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    /** Number of the card. Must be unique across all cards. */
+    @Column(name = "number")
+    private String number;
```

Comment:
Card number should have validation for format (not just uniqueness)
do not forget to implement it during Controller/Service implementation